django_project/certification/apps.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import unicode_literals
```

Comment:
Don't forget to add: ```# coding=utf-8```
You can make a template to make your life easier, like this:

<img width="1042" alt="screen shot 2017-03-24 at 07 44 24" src="https://cloud.githubusercontent.com/assets/831865/24275928/bfae8d3c-1065-11e7-86b2-69e56a96b53f.png">